Lib/test/test_operator.py

```diff
@@ -531,27 +531,33 @@ def test_concat(self):
         self.assertEqual(data2, 'ab')  # must not change
 
         # Subclasses:
-        class ListSubclass(list): ...
+        class ListSubclass(list):
+            pass
 
         data1 = ListSubclass([1, 2])
         data2 = ListSubclass(['a', 'b'])
-        self.assertEqual(operator.concat(data1, data2),
-                         ListSubclass([1, 2, 'a', 'b']))
-        self.assertEqual(operator.concat(data1, data2), data1 + data2)
+
+        res = operator.concat(data1, data2)
+        self.assertIsInstance(res, list)
```

Comment:
This doesn't prove anything, does it? Whether the result is a ListSubclass or a plain list, this will always be true. If you want to say something interesting here I'd assert that it _isn't_ a ListSubclass instance.